README.md

```diff
@@ -113,17 +113,36 @@ optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code Mi
 
 ### Run generation using VLMPipeline API in Python
 
+See [Visual Language Chat](https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/python/visual_language_chat) for a demo application.
+
+Run the following command to download a sample image:
+
+```sh
+curl -O "https://storage.openvinotoolkit.org/test_data/images/dog.jpg"
+```
+
 ```python
+import numpy as np
+import openvino as ov
 import openvino_genai as ov_genai
-#Will run model on CPU, GPU is a possible option
-pipe = ov_genai.VLMPipeline("./MiniCPM-V-2_6/", "CPU")
-rgb = read_image("cat.jpg")
-print(pipe.generate(prompt, image=rgb, max_new_tokens=100))
+from PIL import Image
+
+# Choose GPU instead of CPU in the line below to run the model on Intel integrated or discrete GPU
+pipe = ov_genai.VLMPipeline("/home/ubuntu/models/MiniCPM-V-2_6", "CPU")
+config = ov_genai.GenerationConfig()
```

Comment:
Config creation was intentionally avoided to preserve default model generation config.

Reply:
The current example sets max_new_tokens in the generate() call but that returns with an error: 
```
ValueError: 'max_new_tokens' is unexpected parameter name. Use help(openvino_genai.VLMPipeline.generate) to get list of acceptable parameters.
```
So what is the best practice for adding max_new_tokens without adding a config?

Reply:
@popovaan, please take a look why `max_new_tokens` isn't accepted by `VLMPipeline`

Reply:
looks like after we have migrated to new Sampler, we have not updated a list of generation parameters accepted by Python. So, all manual parameters are ignored and we have to fix it in 2024.5

Reply:
Checked with 2024.5 RC4 that setting max_new_tokens directly works now so I removed the generation config from the example.